kubernetes-kotlin/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Pulumi should support running preinstalled `gradle` if the `./gradlew` and `gradle/wrapper` files are omitted. We currently omit them in the `gradle` template. I'm curious what is the better default for users, do you believe including gradle wrappers is the better default (in which case we should consider adding that to the plain `gradle` template also?

Reply:
yes including the gradlew if is consider better practice in gradle because it allows not to make assumptions about what the user has or not available in the path and also works better with CI, because once run it will be cached under the user profile and it will not pull again.
More detail explanation at: https://tomgregory.com/what-is-the-gradle-wrapper-and-why-should-you-use-it/

Reply:
Agree with @luistrigueiros including gradlew is considered the best practice